src/main/java/intro/bookservice/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package intro.bookservice.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
```

Comment:
It's also good to validate max and min length of text fields. Min length to not allow empty strings and max length to not exceed DB column length restrictions. You can use `Size` annotation for this purpose.